fs/vfs/fs_uio.c

```diff
@@ -51,18 +51,69 @@ ssize_t uio_total_len(FAR const struct uio *uio)
 {
   const struct iovec *iov = uio->uio_iov;
   int iovcnt = uio->uio_iovcnt;
+  size_t offset_in_iov = uio->uio_offset_in_iov;
   size_t len = 0;
```

Comment:
change ssIze_t

Reply:
it can be ssize_t, sure.

otoh, writing code with signed arithmetic uses my brain cycles more.
and from my experience, sometimes compilers produce less efficient machine code.

let me think a bit.